crates/reporting/src/error/parse.rs

```diff
@@ -4491,6 +4480,8 @@ fn to_unfinished_ability_report<'a>(
 enum Next<'a> {
     Keyword(&'a str),
     // Operator(&'a str),
+    // TODO ask Folkert for unused fields str and char
+    #[allow(dead_code)]
     Close(&'a str, char),
```

Comment:
Can you take a look at this @folkertdev? Perhaps these fields are useful when debugging?

Reply:
this type is used for generating better error messages. it looks like none use this information right now, but it seems plausible to me that a message will in the future. I think a `allow(dead_code)` is the way to go here.